database/testapp/src/common_main.cc

```diff
@@ -14,6 +14,7 @@
 
 #include <algorithm>
 #include <ctime>
+
```

Comment:
Since you're not really touching this file, can you just revert this change completely?